### PR TITLE
Mixin to customize the icon

### PR DESCRIPTION
**Pull request by victorhugom**
The  icon (card cvc) inside `gold-cc-cvc-input` element is misplaced when a border is set to the `--paper-input-container-input`.

This change will allow the developer to customize the icon and not just the `Polymer.PaperInputContainer`

### CODE REVIEW

**Comment by victorhugom** [comment]
I signed it!

File: gold-cc-cvc-input.html [hunk 1]
```diff
@@ -63,8 +67,9 @@
         display: block;
       }
 
-      iron-icon {
+      .cvc-icon {
```
Comment (notwaldorf) [inline gold-cc-cvc-input.html]: Why is this change needed?

Comment (victorhugom) [inline gold-cc-cvc-input.html]: For some reason when I put the mixin inside the `iron-icon` tag with the shadow dom on the mixin wasn`t applied, so I put a class to fix this behavior.

Comment (notwaldorf) [inline gold-cc-cvc-input.html]: I've just tested it with the mixin directly on iron-icon, and with this in the demo:
```
--gold-cc-cvc-input-icon: {
      padding-left: 40px;
      background: red;
    }
```

And it seems to work fine:
screen shot 2016-04-15 at 1 56 17 pm

Please give it another try?

Comment (victorhugom) [inline gold-cc-cvc-input.html]: I tested again and works fine

**Comment by notwaldorf** [comment]
Thanks!
